test/with-config.test.js

```diff
@@ -0,0 +1,25 @@
+import { test } from 'tap'
+import Fastify from 'fastify'
+
+import slowDownPlugin from '../index.js'
+import { HEADERS } from '../lib/constants.js'
+
+const OPTIONS = {
+  delay: '1 second',
+  delayAfter: 1,
+  timeWindow: '1 minute'
+}
+
+test('Should work with provided options', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin, OPTIONS)
+  t.teardown(() => fastify.close())
+
+  fastify.get('/', async () => 'Hello fastify-slow-down!')
+
+  let res = await fastify.inject('/')
```

Comment:
why assign a variable here when you don't use it? just don't assign it and create a const on the following line. avoid mutable state